package.json

```diff
@@ -73,11 +73,11 @@
     "taskcluster-client-web": "8.1.1",
     "taskcluster-lib-scopes": "10.0.2",
     "webpack": "4.30.0",
-    "webpack-cli": "3.3.2"
+    "webpack-cli": "3.3.2",
+    "@neutrinojs/eslint": "9.0.0-rc.1",
+    "@neutrinojs/jest": "9.0.0-rc.1"
```

Comment:
Ah, ok - that explains why I saw an eslint error about moving them to dependencies. Thanks @edmorley!

Reply:
@edmorley so I switched to the inline require style as mentioned above: 
```
process.env.NODE_ENV === 'development' &&
      // eslint-disable-next-line global-require
      require('@neutrinojs/eslint')({
        eslint: {
          // We manage our lint config in .eslintrc.js instead of here.
          useEslintrc: true,
        },
      })
```
but I'm still seeing the eslint error: '@neutrinojs/eslint' should be listed in the project's dependencies, not devDependencies` (same for jest). Any ideas?

Reply:
I'd add `import/no-extraneous-dependencies` (presuming that's the rule that's failing) to the `eslint-disable-next-line`. That ESLint rule (from https://github.com/benmosher/eslint-plugin-import) isn't smart enough to know that the conditionals before the import mean it's ok for it to be in `devDependencies` (which is fair enough, it would be too much for the rule to handle every case).

Reply:
Will do - thanks!